config/settings.yml

```diff
@@ -0,0 +1,12 @@
+dor_services:
+  url: 'https://fakeuser:fakepass@fakeserver.example.edu/dor'
```

Comment:
why do we even need a dor-services url?   does workflow require it?

Reply:
Because `dor-workflow-services` uses it.